test/unit/default-transit-auth-method.bats

```diff
@@ -49,40 +49,197 @@ load _helpers
      [ "${actual}" = "default" ]
 }
 
-@test "defaultTransitAuthMethod/CR: settings can be modified" {
+@test "defaultTransitAuthMethod/CR: default settings for auth method" {
     cd `chart_dir`
     local object=$(helm template \
         -s templates/default-transit-auth-method.yaml  \
         --set 'controller.manager.clientCache.persistenceModel=direct-encrypted' \
-        --set 'controller.manager.clientCache.storageEncryption.namespace=foo-ns' \
-        --set 'controller.manager.clientCache.storageEncryption.serviceAccount=foo-sa' \
-        --set 'controller.manager.clientCache.storageEncryption.mount=foo-mount' \
-        --set 'controller.manager.clientCache.storageEncryption.role=foo-role' \
-        --set 'controller.manager.clientCache.storageEncryption.tokenAudiences={vault,foo}' \
-        --set 'controller.manager.clientCache.storageEncryption.keyName=foo-keyName' \
-        --set 'controller.manager.clientCache.storageEncryption.transitMount=foo-transit-mount' \
-        --set 'controller.manager.clientCache.storageEncryption.vaultConnectionRef=foo' \
+        --set 'controller.manager.clientCache.storageEncryption.enabled=true' \
+        . | tee /dev/stderr)
+
+    local actual=$(echo "$object" | yq '.metadata.name' | tee /dev/stderr)
+     [ "${actual}" = "release-name-vault-secrets-operator-default-transit-auth" ]
```

Comment:
super nit: seems like we have some extra whitespace here, would it be possible to reformat it to line up with the line above? Typically shell uses 2 space indentation.